src/components/textfield/textfield.component.tsx

```diff
@@ -48,7 +48,7 @@ export const TextField = (componentProps: ITextFieldProps): React.ReactElement =
   const _onChange = (event: React.FormEvent<HTMLInputElement>) => {
     setCharCountState(event.currentTarget.value.length);
     setPristineState(false);
-    onChange?.call(event.target, event);
+    onChange && onChange(event);
```

Comment:
tsc was complaining about this, haven't properly validated if this still works though.